cpp/ps0003/p03.cpp

```diff
@@ -4,11 +4,17 @@ using namespace std;
 
 int main()
 {
-  int x=0;
-  cin>>x;
- if (x%2==0)
-    cout<< "Ever"<<endl;
- else
-     cout<< "Odd"<<endl;
+  // prompting user for number 2/2
+  int x = 0;
+  cin >> num;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number